### DISCUSSION TITLE
为什么总是拒绝访问呀

### DISCUSSION THREAD

**Discussion by namic00** [spencerwooo/onedrive-vercel-index#710] [Q&A] [answered]
https://myweb-namic00.vercel.app/zh-CN/onedrive-vercel-index-oauth/step-1/
![image](https://user-images.githubusercontent.com/70280045/180030031-1c09c2b7-1dbe-4d4a-86a3-21e87d8fa2e8.png)

**Comment by spencerwooo** [comment, marked as answer]
首先，这一步骤只需要你复制地址栏中的 URL 即可，无法访问是预期现象。其次，可以看到你 URL 中包括 `invalid_client` 的报错信息，说明配置文件或你的登录账户有问题。
